scss/_main.scss

```diff
@@ -40,6 +40,7 @@ summary {
   max-width: 1024px;
   margin-left: auto;
   margin-right: auto;
+  padding: 0 30px 0 30px;
```

Comment:
I want to make it clear to next person who is checking my code.
Usually I will use grunt or gulp to optimize it.
By the way in every single IDE they aslo have extension to helps optimize CSS or JS as well.